examples/readiness-probe-pod

```diff
@@ -0,0 +1,19 @@
+# Copy of pod.yaml without file extension for test
```

Comment:
Move under the e2e test data directory if you're copying it? Also, please add a .yaml file extension.